crates/server/src/api/debug/mod.rs

```diff
@@ -18,18 +35,144 @@ impl Config {
     pub fn into_router(self) -> Router {
         Router::new()
             .route("/packages", get(list_package_names))
+            .route("/package/:name", get(get_package_info))
```

Comment:
Nit: while name is pretty synonymous with package id, perhaps this just for clarity?

```suggestion
            .route("/package/:package_id", get(get_package_info))
```

Reply:
Oops, I even changed it in the function signature...